ee/vellum_ee/workflows/display/nodes/vellum/final_output_node.py

```diff
@@ -47,6 +47,14 @@ def serialize(self, display_context: WorkflowDisplayContext, **kwargs: Any) -> J
             "display_data": self.get_display_data().dict(),
             "base": self.get_base().dict(),
             "definition": self.get_definition().dict(),
+            "outputs": [
+                {
+                    "id": str(self._get_output_id()),
+                    "name": "node_input",
+                    "type": inferred_type,
+                    "value": self._create_workflow_value_descriptor(),
```

Comment:
ah, I think the disconnect is that we should be able to get this value directly from `self.Outputs.value` and serialize that descriptor instead of creating a new one to track